### PR TITLE
chore(docs): some instructions for scripted tests

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala [hunk 1]
```diff
@@ -48,8 +50,17 @@ object ValueEntitySourceGenerator {
...
+         |class ${entity.fqn.name}(val entityId: String) /* extends Abstract${entity.fqn.name} */ {
          |  ${Format.indent(methods, 2)}
+         |
+         |  // TODO override
+         |  def emptyState =
+         |    throw new UnsupportedOperationException("Not implemented yet, replace with your empty entity state")
+         |}
+         |object ${entity.fqn.name} {
+         |  def apply(context: Unit /* ValueEntityContext */):${entity.fqn.name} =
+         |   new ${entity.fqn.name}("TODO")
```
Comment (octonato) [inline codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala:63]: This commit is unrelated.
I guess you were working on another PR that will bring that change already, correct?

Comment (patriknw) [reply codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala:63]: and probably the reason for ci failure

**Review by patriknw — APPROVED** [review]
LGTM after reverting ValueEntitySourceGenerator.scala
Thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,3 +57,15 @@ Ensure to remove/update generated files under `src` if they cause problems.
...
 
 Ensure to remove/update generated files under `src` if they cause problems.
+
+## Interactively run sbt scripted tests
+
+There are a number of Scala projects being tested as 'scripted' tests. After a
+scripted test has completed (successfully or not) it removes the temporary
+directory where it was running.
+
+To interactively inspect the project, you can insert a `$ pause` command in the
```
Comment (patriknw) [inline CONTRIBUTING.md:67]: I tried
```
> compile
$ pause
```
but that will of course not work, since `compile` fails first.

Can we add clarification that it can be scripted like:
```
> protocGenerate
> generateUnmanaged
$ pause
```
or whatever you recommend instead

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -57,3 +57,16 @@ Ensure to remove/update generated files under `src` if they cause problems.
...
+There are a number of Scala projects being tested as 'scripted' tests. After a
+scripted test has completed (successfully or not) it removes the temporary
+directory where it was running.
+
+To interactively inspect the project, you can insert a `$ pause` before the
+first failing command in the `test` script of a scripted test. You can then
+inspect the project, and enter it interactively with 
+`cd /tmp/sbt_431151; sbt -Dplugin.version=....`, specifying the snapshot
+version of the project. Adding `> protocGenerate` or `> generateUnmanaged`
+before `$ pause` might save entering sbt yourself a few times.
```
Comment (patriknw) [inline CONTRIBUTING.md:72]: ah, ofc the `pause` should be added at the top (stupid thinking of me)

**Review by patriknw — APPROVED** [review]
LGTM
